pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -57,6 +57,16 @@ def set_variable(self):
         """
         return self._set_variable
 
+    @property
+    def modified_args(self):
+        """
+        Return an iterable of all the arguments which may be modified by this function.
+
+        Return an iterable of all the arguments which may be modified by this function.
+        This is notably useful in order to determine the constness of arguments.
+        """
+        return (self._set_variable,)
```

Comment:
I may have misunderstood the meaning of `modified_args`.

When we call any method on a set, the set itself is never modified. So why do we not return an empty tuple `()` here?